_includes/v20.1/faq/sql-query-logging.md

```diff
@@ -18,23 +19,91 @@ SQL audit logging is useful if you want to log all queries that are run against
 
 For production clusters, the best way to log all queries is to turn on the [cluster-wide setting](cluster-settings.html) `sql.trace.log_statement_execute`:
 
+{% include copy-clipboard.html %}
 ~~~ sql
 > SET CLUSTER SETTING sql.trace.log_statement_execute = true;
 ~~~
 
 With this setting on, each node of the cluster writes all SQL queries it executes to a secondary `cockroach-sql-exec` log file. Use the symlink `cockroach-sql-exec.log` to open the most recent log. When you no longer need to log queries, you can turn the setting back off:
 
+{% include copy-clipboard.html %}
 ~~~ sql
 > SET CLUSTER SETTING sql.trace.log_statement_execute = false;
 ~~~
 
-Another useful cluster setting is `sql.log.slow_query.latency_threshold`, which is used to log only queries whose service latency exceeds a specified threshold value (e.g., 100 milliseconds):
+Log files are written to CockroachDB's standard [log directory](debug-and-error-logs.html#write-to-file).
+
+### Slow query logs
+
+Another useful [cluster setting](cluster-settings.html) is `sql.log.slow_query.latency_threshold`, which is used to log only queries whose service latency exceeds a specified threshold value (e.g., 100 milliseconds):
 
+{% include copy-clipboard.html %}
 ~~~ sql
 > SET CLUSTER SETTING sql.log.slow_query.latency_threshold = '100ms';
 ~~~
 
-Each node that serves as a gateway will then record slow SQL queries to a `cockroach-sql-slow` log file. Use the symlink `cockroach-sql-slow.log` to open the most recent log. For more details on logging slow queries, see [Slow query log](query-behavior-troubleshooting.html#slow-query-log).
+Each node that serves as a gateway will then record slow SQL queries to a `cockroach-sql-slow` log file. Use the symlink `cockroach-sql-slow.log` to open the most recent log. For more details on logging slow queries, see [Using the slow query log](query-behavior-troubleshooting.html#using-the-slow-query-log).
+
+Log files are written to CockroachDB's standard [log directory](debug-and-error-logs.html#write-to-file).
+
+### Authentication logs
```

Comment:
Please emphasize somewhere in this section that both the auth logs (like the SQL audit logs) perform one disk I/O per event so they should not be enabled by default nor casually unless the user understands the performance+cost impact.

(This is true because their primary purpose is to guarantee non-repudiability of events and they synchronize to disk all the time - but I'm not sure that's worth explaining).

Reply:
Added a line to this effect!